test/RedisStore-test.js

```diff
@@ -50,6 +51,15 @@ describe('RedisStore', () => {
     util.ensureKeyspaceNotifications.calledOnce.should.be.ok();
   });
 
+  it('should quit', done=>{
```

Comment:
nit: `done => {` we should really just bring in [Prettier](https://github.com/prettier/prettier) and never have to talk about spacing again.

Reply:
I agree. I like how the trend has moved toward automating such.